sql/core/src/test/scala/org/apache/spark/sql/execution/ShufflePartitionsUtilSuite.scala

```diff
@@ -36,7 +36,10 @@ class ShufflePartitionsUtilSuite extends SparkFunSuite {
       Seq.fill(mapOutputStatistics.length)(None),
       targetSize,
       minNumPartitions)
-    assert(estimatedPartitionStartIndices.forall(_ === expectedPartitionStartIndices))
```

Comment:
the previous assert has no effect if the result estimatedPartitionStartIndices is empty